### PR TITLE
Fix: update 1-projection.md description

### PR DESCRIPTION
**Pull request by dmmishchenko**
## Description 
This PR fixed constraints section for angular challenge ["Projection"](https://angular-challenges.vercel.app/challenges/angular/1-projection/)

## Related issues:
[#238](https://github.com/tomalaforge/angular-challenges/issues/238)

### CODE REVIEW

File: docs/src/content/docs/challenges/angular/1-projection.md [hunk 1]
```diff
@@ -32,6 +32,6 @@ While the application works, the developer experience is far from being optimal.
 ## Constraints
 
 - You <b>must</b> refactor the `CardComponent` and `ListItemComponent`.
-- The `NgFor` directive must be declared and remain inside the `CardComponent`. You might be tempted to move it to the `CardComponent` but avoid doing so.
+- The `NgFor` directive must be declared and remain inside the `CardComponent`. You might be tempted to move it to the `ListItemComponent` but avoid doing so.
```
Comment (tomalaforge) [inline docs/src/content/docs/challenges/angular/1-projection.md:35]: Indeed that's a mistake, but that's not the `ListItemComponent`, you must be tempted to move it to the `ParentCardComponent` like `TeacherCardComponent`, ...

Comment (dmmishchenko) [reply docs/src/content/docs/challenges/angular/1-projection.md]: Fixed, thank you for your feedback:
Screenshot 2023-10-29 at 16 31 21

**Comment by tomalaforge** [comment]
@allcontributors, add @dmmishchenko for doc

**Comment by dmmishchenko** [comment]
Hi @tomalaforge ,
Do i need to perform something else to improve this PR?

**Comment by tomalaforge** [comment]
@all-contributors add @dmmishchenko  doc

**Review by tomalaforge — APPROVED** [review]

**Pull request merged**